server/src/migrations/Migration20210220195842.ts

```diff
@@ -0,0 +1,10 @@
+import { Migration } from '@mikro-orm/migrations'
```

Comment:
this file looks like it got formatted by Prettier

Reply:
fixed in latest commit.